torch/ao/quantization/quantizer/xnnpack_quantizer_utils.py

```diff
@@ -825,6 +825,12 @@ def _annotate_add(
     for add_partition in add_partitions:
         annotated_partitions.append(add_partition.nodes)
         add_node = add_partition.output_nodes[0]
+        for output_node in add_partition.output_nodes:
+            # constants may be propagated through partition as an
+            # output, so we need to check that output node is a call_function
+            if output_node.op == "call_function":
```

Comment:
I see, this could be true for mul patterns as well. thoughts on refactoring the matching code to use `aten.add`/`aten.add_` ops? cc @kimishpatel